coffee_core/src/coffee.rs

```diff
@@ -14,13 +14,14 @@ use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 use tokio::process::Command;
 
+use chrono::{TimeZone, Utc};
```

Comment:
this should be under the tokio import. We try to keep separate the our import and the library import.